CONTRIBUTING.md

```diff
@@ -66,14 +72,40 @@ Pull requests should have a title that follows the specification, otherwise, mer
 - `test: ` prefix in the title indicates that PR is only related to tests and there is no need to trigger release.
 - `refactor: ` prefix in the title indicates that PR is only related to refactoring and there is no need to trigger release.
 
-What about MAJOR release? just add `!` to the prefix, like `fix!: ` or `refactor!: `
+What about a MAJOR release? Just add `!` to the prefix, like `fix!: ` or `refactor!: `
```

Comment:
We don't have any release process inside the website. Do we need this point?

Reply:
yup, you;re right - will remove once we also clarify other comments